translate/storage/lisa.py

```diff
@@ -327,7 +327,7 @@ def serialize(self, out=None):
         """Converts to a string containing the file's XML"""
         root = self.document.getroot()
         if self.XMLdoublequotes:
-            out.write(b'<?xml version="1.0" encoding="utf-8"?>\n')
+            out.write(b'<?xml version="1.0" encoding="UTF-8"?>\n')
```

Comment:
This is definitely not a desired change for resx (see https://github.com/WeblateOrg/weblate/issues/2807) and probably also for Android. So either stick with lowercase or make it configurable per file format.

Reply:
When using xmllint (http://xmlsoft.org/xmllint.html), the lowercase is reported as an error ; so I consider the UTF-8 should be prefer for xml. Having it configurable could be a solution indeed.

I tried to add the implementation, first time using python so I'm not sure about it.

So I need to set the new constant to false for resx and Android (what format should I look at ?). Is there any other ?